needles/control_center_about_capplet.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "control_center",
+    "control_center_about_capplet"

```

Comment:
Don't you need to cut a square out of this for the version number, which changes with every ostree update? Or is the match "good enough"?

Reply:
There is a degree of fuzziness in the matching (II think it is happy with a 90%+ or 95%+ match?). I’d rather wait until this actually fails and then add an appropriate exclusion area, rather than pre-empt the failure.